bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.0.5.html

```diff
@@ -44,7 +44,11 @@ <h1>{{ _('You’ve got the latest browser. Now get the rest of Firefox.') }}</h1
             }}
           {% endblock %}
         </div>
-        <a href="{{ url('firefox.accounts') }}">{{ _('Learn more about joining Firefox') }}</a>
+        {% if LANG.startswith('en-') %}
```

Comment:
Since we're updating strings anyway, let's go ahead and remove this conditional.